doc/api-changes/correct_root_2016-08-15.rst

```diff
@@ -0,0 +1,6 @@
+`correct_root` supports uids
```

Comment:
(I suggest just adding one bullet point until "Internal API Changes" with all of these facts in it.